extensions/blocks/podcast-player/components/header.js

```diff
@@ -5,31 +5,29 @@ import { memo } from '@wordpress/element';
 
 const Header = memo(
 	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription } ) => (
-		<div className="jetpack-podcast-player__header-wrapper">
-			<div className="jetpack-podcast-player__header" aria-live="polite">
-				{ showCoverArt && cover ? (
-					<div className="jetpack-podcast-player__track-image-wrapper">
+		<div className="jetpack-podcast-player__header">
+			<div className="jetpack-podcast-player__cover-wrapper" aria-live="polite">
+				{ showCoverArt && cover && (
+					<div className="jetpack-podcast-player__cover">
 						{ /* alt="" will prevent the src from being announced. Ideally we'd have a cover.alt, but we can't get that from the RSS */ }
-						<img className="jetpack-podcast-player__track-image" src={ cover } alt="" />
+						<img src={ cover } alt="" />
```

Comment:
Why not `.jetpack-podcast-player__cover-image`? I understand the markup is most likely always going to be an `<img>` but either way I don't see a downside to decoupling the styles from the markup.

Reply:
Done in https://github.com/Automattic/jetpack/pull/15211/commits/7f64da8ebba98b4e8c471b68145fe89ce5626ca6